### PR TITLE
Docs: Clean up contribute docs

### PR DESCRIPTION
**Pull request by marcusolsson**
**What this PR does / why we need it**:

This pull request contains minor clean ups the contribution docs. Fixes paths, and removes duplicated content, etc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,19 +54,12 @@ Unsure where to begin contributing to Grafana? Start by browsing issues labeled
...
 
 Before we can accept your pull request, you need to [sign our CLA](https://grafana.com/docs/contribute/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
 
-## Community
-
-- Follow [@grafana on Twitter](https://twitter.com/grafana/)
-- Read and subscribe to the [Grafana blog](https://grafana.com/blog/)
-- If you have a specific question, check out our [discussion forums](https://community.grafana.com).
-- For general discussions, join us on the [official Slack](http://slack.raintank.io/).
-
```
Comment (daniellee) [inline CONTRIBUTING.md]: Why remove this section completely? Is getting rewritten and added somewhere else?

Comment (marcusolsson) [reply CONTRIBUTING.md]: I added it to the README.md, under "Get involved". Is it worth duplicating?

Comment (daniellee) [reply CONTRIBUTING.md]: That's fine - just wondering.

**Review by daniellee — APPROVED** [review]
LGTM

**Comment by daniellee** [comment]
@marcusolsson there is a merge conflict with a deleted file to resolve but think you can merge once you have resolved that.

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**